packages/discovery-react-components/src/utils/document/__tests__/transformEnrichment.spec.tsx

```diff
@@ -0,0 +1,82 @@
+import transformEnrichment from '../transformEnrichment';
+import isEqual from 'lodash/isEqual';
+import contractData from 'components/CIDocument/components/CIDocument/__fixtures__/contract.json';
+import invoiceData from 'components/CIDocument/components/CIDocument/__fixtures__/invoice-index_op.json';
+import purchaseOrderData from 'components/CIDocument/components/CIDocument/__fixtures__/po-index_op.json';
+import { EnrichedHtml } from 'components/CIDocument/types';
+
+describe('transformEnrichment', () => {
```

Comment:
having this describe block mention that it's used for contract data would make this a bit more readable to me, since there are several `transformEnrichment` blocks. Maybe you could tweak the other blocks as well to mention the type of data the test's relate to